### PR TITLE
Gradle: Replacing backslashes with slashes on Windows

### PR DESCRIPTION
**Pull request by bitsandfoxes**
The previous PR #507 missed the symbols path. Also, for consistency in the `gradle.build`file I opted to replace Windows path backslashes with slashes (the hardcoded paths already use slashes)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -110,34 +110,34 @@ jobs:
...
+        uses: bitsandfoxes/setup-unity@465111b8a4184a1ee4549e26cae0638b0df34f48
         id: setup-unity
         with:
           unity-version: ${{ matrix.unity-version }}
           unity-version-changeset:  ${{ matrix.unity-version-changeset }}
 
       - name: Setup Unity Modules - Windows IL2CPP
         if: matrix.os == 'windows-latest'
-        uses: kuler90/setup-unity@f44e3f35ce3737316a15321ec3468161940e23e8
+        uses: bitsandfoxes/setup-unity@465111b8a4184a1ee4549e26cae0638b0df34f48
```
Comment (lucas-zimerman) [inline .github/workflows/ci.yml:121]: Is this change related to this PR? I don't follow why this change was made😅

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -110,34 +110,34 @@ jobs:
           key: ${{ matrix.os }}-${{ hashFiles('submodules-status') }}-${{ matrix.unity-version }}-${{ matrix.unity-version-changeset }}-${{ hashFiles('package/package.json') }}
 
       - name: Setup Unity
-        uses: kuler90/setup-unity@f44e3f35ce3737316a15321ec3468161940e23e8
+        uses: bitsandfoxes/setup-unity@465111b8a4184a1ee4549e26cae0638b0df34f48
```
Comment (bruno-garcia) [inline .github/workflows/ci.yml:113]: It's best that we have this fork on `getsentry`

File: src/Sentry.Unity.Editor/Android/DebugSymbolUpload.cs [hunk 3]
```diff
@@ -26,9 +26,12 @@ public static string GetSymbolsPath(string unityProjectPath, string gradleProjec
         public static void AppendUploadToGradleFile(string sentryCliPath, string gradleProjectPath, string symbolsDirectoryPath, IApplication? application = null)
         {
             application ??= ApplicationAdapter.Instance;
+
+            // On Windows the paths contains backslashes that need to be either escaped or replaced with slashes (opting for slash for consistency)
```
Comment (bruno-garcia) [inline src/Sentry.Unity.Editor/Android/DebugSymbolUpload.cs:30]:
```suggestion
            // Gradle doesn't support backslahes on path (Windows) so converting to forward slashes
```

**Review by bruno-garcia — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bitsandfoxes) [reply .github/workflows/ci.yml:121]: Not related to this PR, superseded by https://github.com/getsentry/sentry-unity/pull/510
